### PR TITLE
github: autolabel pull requests

### PR DESCRIPTION
**Pull request by teto**
depending on which files are modified

### CODE REVIEW

**Review by vigoux — COMMENTED** [review]
Don't you also need to add the labeler action in order to have that working ?

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
```
Comment (jamessan) [inline .github/workflows/labeler.yml:4]: Need to add

```
permissions:
  contents: read
  pull-requests: write
```

so the auto-generated token can set labels on the PRs.

Comment (teto) [reply .github/workflows/labeler.yml:4]: first time setting up GHA actions, where did you see this ? it's not shown in the example https://github.com/marketplace/actions/labeler#create-workflow

**Review by mhinz — APPROVED** [review]

**Review by vigoux — APPROVED** [review]

**Comment by teto** [comment]
@vigoux is that your way of saying I should add

```
"ci":
  - .github/**/*
 ```
:p ? maybe in a follow up MR.

**Comment by vigoux** [comment]
Not sure we should do it, ci PRs aren't really made that often, so...

**Comment by mhinz** [comment]
I think `ci` would actually make sense, but let's wait a bit for now and see how it works. Maybe we find other things (to add or remove again) as well.

I'll be waiting for the first PR that changes the header in a lot of files or moves them around, leading to the.. _labelcalypse_!

**Pull request merged**